src/docs/cmdstan-guide/getting-started.tex

```diff
@@ -181,11 +177,8 @@ \subsection{Data Set}
 the \code{stan\_rdump} in package \code{rstan}.
 %
 \footnote{
-The data is also available from the \code{example-models} repository
-on GitHub,
-\url{https://github.com/stan-dev/example-models/blob/master/basic_estimators/bernoulli.data.R}.
-It is also included with the CmdStan distribution and can be found at path
-\nolinkurl{stan/example-models/basic_estimators/bernoulli.data.R}.
+The data is also included with the CmdStan distribution and can be found at path
+\nolinkurl{stan/example-models/basic_estimators/bernoulli.data.R} and \nolinkurl{stan/example-models/basic_estimators/bernoulli.data.json} .
```

Comment:
This should just be `\nolinkurl{examples/bernoulli/bernoulli.data.R} and \nolinkurl{examples/bernoulli/bernoulli.data.json}` I think